### ISSUE TITLE
Substrate Wallet Selector component

### ISSUE THREAD

**Issue by mpetrunic** [sygmaprotocol/sygma-widget#73]
## Implementation details

The substrate ecosystem doesn't have ready-made framework-agnostic UI component for selecting the wallet. It should pop simple modal listing all available extension wallets + wallet connect. Assets for displaying wallets: https://polkadot.cloud/extensions

**Comment by mpetrunic** [comment]
Inspo:
![image](https://github.com/sygmaprotocol/sygma-widget/assets/8836210/7e026dc1-a8e4-41ce-b1b4-ba96d812eeda)
![image](https://github.com/sygmaprotocol/sygma-widget/assets/8836210/c10de34c-657f-4c56-82a3-c2850f1e2685)

**Comment by sweetpea22** [comment]
<img width="1237" alt="Screen Shot 2024-02-05 at 11 03 39 AM" src="https://github.com/sygmaprotocol/sygma-widget/assets/47398578/ba29b5d5-a958-4967-958e-8470bb092804">
<img width="1246" alt="Screen Shot 2024-02-05 at 11 03 50 AM" src="https://github.com/sygmaprotocol/sygma-widget/assets/47398578/664e6cbd-e40a-4f2b-898a-987f98b1f44b">

ok added some sketches. check it [here](https://www.figma.com/file/T3A0opRIH07hSqUaESKgFV/Sygma-Widget-UI?type=design&node-id=406%3A2940&mode=design&t=G1aqX2ZGPll0MCAw-1)

**Comment by mpetrunic** [comment]
I don't think we need loading screen, nothing will happen if user clicks connect twice though. For MVP I would not go into separate screens for selecting account
